openmmtools/tests/test_integrators.py

```diff
@@ -206,6 +206,39 @@ def test_vvvr_shadow_work_accumulation():
     assert('shadow_work' not in names_of_globals)
 
 
+def test_vvvr_protocol_work_accumulation():
+    """When `measure_protocol_work==True`, assert that global `protocol_work` is initialized to zero and
+    reaches a zero value after integrating a few dozen steps without perturbation.
+
+    By default (`measure_protocol_work=False`), assert that there is no global name for `protocol_work`."""
+
+    testsystem = testsystems.HarmonicOscillator()
+    system, topology = testsystem.system, testsystem.topology
+    temperature = 298.0 * unit.kelvin
+    integrator = integrators.VVVRIntegrator(temperature, measure_protocol_work=True)
+    context = openmm.Context(system, integrator)
+    context.setPositions(testsystem.positions)
+    context.setVelocitiesToTemperature(temperature)
+    assert(integrator.getGlobalVariableByName('protocol_work') == 0), "Protocol work should be 0 initially"
+    integrator.step(25)
+    assert(integrator.getGlobalVariableByName('protocol_work') == 0), "There should be no protocol work."
+    perturbed_K=99.0 * unit.kilocalories_per_mole / unit.angstroms**2
+    context.setParameter('testsystems_HarmonicOscillator_K', perturbed_K)
+    integrator.step(1)
+    assert (integrator.getGlobalVariableByName('protocol_work') != 0), "There should be protocol work after perturbing."
```

Comment:
There are multiple `assert` statements in this functions that each test a different aspect. It's certainly worth considering splitting these into separate tests.